lang/python/python-protobuf/Makefile

```diff
@@ -0,0 +1,45 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-protobuf
+# Must match the version of the `protobuf` package
+PKG_VERSION:=3.17.3
```

Comment:
I wonder, why do we need to use 3.17.3 version? Its from 2021.
See: https://github.com/protocolbuffers/protobuf/releases/tag/v3.17.3

The latest version is currently v29.0. Also, the latest version does not provide Python tarball at all.

Also, this one is not going to work with you https://github.com/meshtastic/python/blob/7c89e231bdf9a784497366fec1c5b27ea729bbbb/pyproject.toml#L12 and it looks weird then you want to use the latest version of protobuf from pypi and then the version is the same with library.

Reply:
@BKPepe I was planning to submit a separate PR updating the `protobuf` and `python-protobuf` package after this is merged.

However, you're correct, in newer versions all protobuf bindings (cpp, python, etc) are bundled together, (ex: [`protobuf-29.1.tar.gz`](https://github.com/protocolbuffers/protobuf/releases/download/v29.1/protobuf-29.1.tar.gz)).
It may be more prudent to update the [`protobuf`](https://github.com/openwrt/packages/blob/master/libs/protobuf/Makefile) package and add the python bindings inline (no need for a separate package, and easier to keep the versions in sync).

Reply:
Yep, please do that.

Reply:
[libgpiod](https://github.com/openwrt/packages/blob/master/libs/libgpiod/Makefile) seems like a decent example for combined cmake + python packages.